src/web/mjs/connectors/TCBScans.mjs

```diff
@@ -45,7 +45,17 @@ export default class TCBScans extends Connector {
     async _getPages(chapter) {
         const uri = new URL(chapter.id, this.url);
         const request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, 'picture > source');
-        return data.map(x => x.src);
+        const data = await this.fetchDOM(request, 'picture > source');
+
+        //test if image really exists
+        const promiseArray = data.map(page=> new Promise((resolve, reject) =>{
```

Comment:
This would flood the website with http requests on images.

Reply:
If you have another way to know if an image exist without trying to download it please tell me. 🤷 

I would add that since they deliberatly add fake additional pictures to thwart Hakuneko downloading, its only a fair game.

Its a HEAD request, not a big deal tho.

Reply:
I just meant to do it with a for loop so that the requests are sequentials and not done in parallel.